tests/components/onboarding/test_views.py

```diff
@@ -277,3 +329,51 @@ async def test_onboarding_core_sets_up_met(hass, hass_storage, hass_client):
 
     await hass.async_block_till_done()
     assert len(hass.states.async_entity_ids("weather")) == 1
+
+
+async def test_onboarding_core_sets_up_rpi_power(
+    hass, hass_storage, hass_client, aioclient_mock, rpi
+):
+    """Test that the core step sets up rpi_power on RPi."""
+    mock_storage(hass_storage, {"done": [const.STEP_USER]})
+    await async_setup_component(hass, "persistent_notification", {})
+
+    assert await async_setup_component(hass, "onboarding", {})
+
+    client = await hass_client()
+
+    with patch(
+        "homeassistant.components.rpi_power.config_flow.new_under_voltage"
+    ), patch("homeassistant.components.rpi_power.binary_sensor.new_under_voltage"):
+        resp = await client.post("/api/onboarding/core_config")
+
+        assert resp.status == 200
+
+        await hass.async_block_till_done()
+
+    rpi_power_state = hass.states.get("binary_sensor.rpi_power_status")
+    assert rpi_power_state
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6243615)